src/Platforms/SqlitePlatform.php

```diff
@@ -34,11 +34,12 @@
 /**
  * The SqlitePlatform class describes the specifics and dialects of the SQLite
  * database platform.
- *
- * @todo   Rename: SQLitePlatform
  */
 class SqlitePlatform extends AbstractPlatform
 {
+    /** @var bool True by default for BC reasons, option to be removed in DBAL 4.0 */
+    private $replaceDotWithTwoUnderscoresInTableName = true;
```

Comment:
```suggestion
    private bool $schemaEmulationEnabled = true;
```

* We can use typed properties in 3.4, so we should use them.
* The removal is documented for the corresponding setter, so I don't think we need to repeat that.
* The property should reflect the (deprecated) feature.